Sample/Assets/Scripts/QRCode.cs

```diff
@@ -4,9 +4,9 @@
 using Microsoft.MixedReality.Toolkit.Input;
 using UnityEngine;
 
-namespace QRTracking
+namespace SampleQRCodes

```

Comment:
While we're here, do we want to prepend `Microsoft.MixedReality.` (and to the asmdef below)?

Reply:
Good point.  Fixed in latest iteration.